pkg/api/v1/types.go

```diff
@@ -924,13 +940,31 @@ type VirtualMachineList struct {
 	Items []VirtualMachine `json:"items"`
 }
 
+// VirtualMachineRunStrategy is a label for the requested VirtualMachineInstance Running State at the current time.
+// ---
+// +k8s:openapi-gen=true
+type VirtualMachineRunStrategy string
+
+// These are the valid VMI running states
+const (
+	RunStrategyRunning        VirtualMachineRunStrategy = "runAlways"
+	RunStrategyHalted         VirtualMachineRunStrategy = "halted"
+	RunStrategyManual         VirtualMachineRunStrategy = "manual"
+	RunStrategyRestartOnError VirtualMachineRunStrategy = "restartOnError"
```

Comment:
I thought we were taking a "job" like approach here and calling this something like "runToCompletion"

https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/